.github/workflows/core-checks.yml

```diff
@@ -40,6 +40,7 @@ jobs:
       - name: Check File Headers
         run: |
           cd inspect
-          ../tools/.github/scripts/check-header.py --json ${HOME}/files_modified.json ${HOME}/files_added.json ${HOME}/files_renamed.json
+          export PYTHONPATH=tools/.github/scripts:${PYTHONPATH}
+          .github/scripts/core_checker.py --json ${HOME}/files_modified.json ${HOME}/files_added.json ${HOME}/files_renamed.json
```

Comment:
Could we add a print statement for the file list ? May be useful to see what the script is run on

Reply:
Added this